.github/workflows/operator-test.yaml

```
@@ -65,7 +65,7 @@ jobs:
           until kubectl get ns opentelemetry-operator-system 2>&1 | grep "namespaces \"opentelemetry-operator-system\" not found"; do sleep 1; done
           kubectl create namespace opentelemetry-operator-system
           helm install --namespace=opentelemetry-operator-system my-opentelemetry-operator ./charts/opentelemetry-operator
-          kubectl wait --timeout=5m --for=condition=available deployment opentelemetry-operator-controller-manager -n opentelemetry-operator-system
+          kubectl wait --timeout=5m --for=condition=available deployment opentelemetry-operator -n opentelemetry-operator-system
```

```suggestion
          kubectl wait --timeout=5m --for=condition=available deployment my-opentelemetry-operator -n opentelemetry-operator-system
```

Since we use fullname now and the chart is installed as `my-opentelemetry-operator` I believe this is the name of the deployment the CI is looking for.

---

Ah yes I've missed it, thank you.

But chart-testing also failing with:

```
Run until kubectl get ns opentelemetry-operator-system 2>&1 | grep "namespaces 

\"opentelemetry-operator-system\" not found"; do sleep 1; done

Error from server (NotFound): namespaces "opentelemetry-operator-system" not found
namespace/opentelemetry-operator-system created
```

Am I missing something?

---

You can ignore that namespace error.  The CI should be taking advantage of helm's --create-namespace feature I think.  I'll handle that in a separate PR.